### DISCUSSION TITLE
initialize lifecycle in activity with java

### DISCUSSION THREAD

**Discussion by khsuzan** [skydoves/Balloon#153] [Q&A] [answered]
Do i need to initialize lifecycleowner into activity . If need, then how pls. I am new bie

**Comment by skydoves** [comment, marked as answer]
Since `1.3.0`, we don't need to call the `setLifecycleOwner` manually anymore.
The `Balloon` initializer will find and set the `lifecycleOwner` from your context.
You can check out the `Balloon` popup don't leak memory when your activity's lifecycle is `onPuase` or `onDestory`.
Thanks :)
